packages/notte-agent/src/notte_agent/common/types.py

```
@@ -65,6 +66,34 @@
+                exclude_fields.remove("selector")
+                exclude_fields.remove("option_selector")
+            exclude_fields.remove("text_label")
+            if "value" in exclude_fields:
+                exclude_fields.remove("value")
+            logger.info(f"Excluding fields: {exclude_fields}")
+            actions.append(step.action.model_dump(exclude=exclude_fields, exclude_none=True))
+            # actions.append(result.action)
+        with open(file_path, "w") as f:
+            json.dump(dict(actions=actions), f, indent=4)
+
+    async def to_workflow(self, variables_format: type[BaseModel] | None = None) -> Workflow:
```

Ruff (0.12.2)

Undefined name `Workflow`

(F821)